src/System.Runtime/tests/System/Reflection/StrongNameKeyPairTests.cs

```diff
@@ -2,66 +2,101 @@
+        [Theory]
+        [MemberData(nameof(Ctor_ByteArray_TestData))]
+        public void Ctor_ByteArray(byte[] keyPairArray)
+        {
+            // Can't validate anything as this is a nop in .NET Core.
+            // Just make sure it doesn't throw.
+            var keyPair = new StrongNameKeyPair(keyPairArray);
+        }
+
+        [Fact]
+        public void Ctor_NullKeyPairArray_ThrowsArgumentNullException()
+        {
+            AssertExtensions.Throws<ArgumentNullException>("keyPairArray", () => new StrongNameKeyPair((byte[])null));
+        }
+
+        [Theory]
+        [MemberData(nameof(Ctor_ByteArray_TestData))]
+        public void Ctor_FileStream(byte[] keyPairArray)
+        {
+            string tempPath = GetTestFilePath();
+            File.WriteAllBytes(tempPath, keyPairArray);
+            using (FileStream fileStream = File.OpenRead(tempPath))
+            {
+                // Can't validate anything as this is a nop in .NET Core.
+                // Just make sure it doesn't throw.
+                var keyPair = new StrongNameKeyPair(fileStream);
+            }
```

Comment:
FWIW, there are things you could validate, like that it throws an exception when given a stream it can't read from, that it reads the expected number of bytes from the stream, etc.